serverlessworkflow/modules/ROOT/pages/cloud/deploying-on-minikube.adoc

```diff
@@ -1 +1,369 @@
+=====================================================
+06-22-2022
+:compat-mode!:
+// Metadata:
+:description: Deploying Serverless Application on Minikube
+:keywords: kogito, workflow, quarkus, serverless, kn, kubectl, minikube
+:table-caption: Data Set
+// links
+:minikube_url: https://minikube.sigs.k8s.io
+:kn_cli_url: https://knative.dev/docs/install/client/
+:kn_cli_quickstart_plugin_url: https://knative.dev/docs/install/quickstart-install/#install-the-knative-cli
+:knative_on_minikube_step_by_step_url: https://redhat-developer-demos.github.io/knative-tutorial/knative-tutorial/setup/minikube.html
+:knative_issue_url: https://github.com/knative/serving/issues/6101
+
+
+This document describes how to deploy a Serverless Application using a local Kubernetes cluster link:{minikube_url}}[Minikube] as well the steps required to get the Knative plataform running. Please make sure to read the Get Started Minikube guide and assure that your machine meets the System Requirement.
+
+.Prerequisites
+
+* Serverless Application ready to use.
+** Please follow this xref:cloud/build-workflow-image-with-quarkus-cli.adoc[guide] to learn how to build the Application Container using your Serverless Application or the example provided.
+* Minikube is installed.
+** kubectl command line tool, if you don't have it installed, Minikube handles it for you.
+* Knative CLI, link:{kn_cli_url}[kn].
+* Optionally, link:{quarkus_cli_url}[Quarkus CLI].
+
+
+== Preparing the Minikube with Knative
+
+.Prerequisites
+* minikube installed and running, link:{minikube_url}/docs/start[start guide].
+* kn cli installed, link:{kn_cli_url}/install-kn//[install guide].
+* kubectl.
+
+.Procedure
+. Check if Minikube is correctly installed, on a terminal type:
++
+[source,shell]
+----
+minikube version
+----
+
+. For Knative CLI:
++
+[source,shell]
+----
+kn version
+----
+
+. For kubectl CLI:
++
+[source,shell]
+----
+kubectl version
+----
+
+[NOTE]
+.kubectl
+====
+If you don't have it installed, execute the following command to let minikube handle it for you:
+[source,shell]
+----
+alias kubectl="minikube kubectl --"
+----
+====
+
+Now, the `kubectl` command should be available.
+
+=== Installing Knative on Minikube
+.Procedure
+. Install and configure Knative on Minikube
++
+Knative CLI offers a plugin called `quickstart` that will configure everything that is needed for you, to install it follow this link:{kn_cli_quickstart_plugin_url}[steps].
++
+With the plugin configured, execute:
++
+[source,shell]
+----
+kn quickstart minikube
+----
++
+NOTE: To install everything manually, there is this cool link:{knative_on_minikube_step_by_step_url}[post] from Red Hat Developers blog.
++
+To open the Minikube WebConsole on the `knative` profile, use the following command
++
+[source,shell]
+----
+minikube dashboard --profile knative
+----
+The command above will open the Kubernetes Management Console on your browser, if that does not open, copy the `url` that was returned.
+
```

Comment:
What about adding a note that every command must be done with --profile knative from now on? Maybe even clarifying that the quickstart adds the knative profile to your minikube installation.

Reply:
maybe the second options is better, because users might want to copy and past these commands.
Will add a small note about the profile that will be created.